tests/performance/plugins/server-timing/mu-plugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+add_action(
+	'template_redirect',
+	function() {
+
+		global $server_timing_values, $timestart, $template_start;
+
+		if ( ! is_array( $server_timing_values ) ) {
+			$server_timing_values = array();
+		}
+
+		$template_start = microtime( true );
+		$server_timing_values['before-template'] = $template_start - $timestart;
+
+		ob_start();
+
+		add_action(
+			'shutdown',
+			function() {
+
+				global $server_timing_values, $timestart, $template_start;
```

Comment:
See above, here you could access the local variables instead of using globals, e.g. like:
```suggestion
			function() use ( $server_timing_values, $template_start ) {
```

Reply:
Thanks for the suggestion.